physical/raft/raft_test.go

```diff
@@ -218,6 +218,14 @@ func compareDBs(t *testing.T, boltDB1, boltDB2 *bolt.DB, dataOnly bool) error {
 	return nil
 }
 
+func TestRaft_BoltMetrics(t *testing.T) {
```

Comment:
Is it that this test was only meant to be used locally for testing and not to actually push?

Reply:
Yep, same as the replace directive above. I was trying to sort out how to test that these metrics are actually being emitted but it's not fully baked yet. I'm not even sure it's a test worth writing, but since the whole point of this PR was emitting these metrics, I figured I should write a test that confirms it's actually happening. Do you have an opinion on this?